### DISCUSSION TITLE
I am  sandip sapkale from India

### DISCUSSION THREAD

**Discussion by Sandipsapkale** [RequestNetwork/requestNetwork#1507] [Introductions]
Internet saving

**Comment by Sandipsapkale** [comment]
If we make an agreement with all the internet companies that the internet balance of your customers will not be used, then we will have an app, your internet balance will remain in the app for 100 days, if you give the company even 1 rupee after every customer, this will be a benefit for the customer. Think how many users there are in the world. Not every person has wifi and even if there is wifi xyz person can take internet from him all these things are possible only if the app maker is smart.

**Reply by MantisClone** [reply]
Hi! Thanks for sharing. I don't understand what you mean or how it relates to Request Network.
